frameworks/cassandra/universe/config.json

```diff
@@ -91,6 +91,11 @@
           "description": "The Marathon-style placement constraint string controlling node placement",
           "default": ""
         },
+        "virtual_network":{
```

Comment:
Right now, yes, because that's the only virtual network on DCOS clusters. In the future the there may be more and we'll template out entire network specs.